src/widget/volume/mod.rs

```diff
@@ -0,0 +1,44 @@
+use std::sync::Arc;
+use widget::base::Widget;
+use format::data::Format;
+use std::sync::mpsc::Sender;
+
+#[cfg(unix)] pub mod alsa;
+#[cfg(unix)] pub use self::alsa::ALSA;
```

Comment:
`#[cfg(target_os = "linux")]`. It's the Advanced *Linux* Sound Architecture after all.